lib/hq/graphql/ext/object_extensions.rb

```diff
@@ -96,7 +96,16 @@ def field_from_column(column, auto_nil:)
             name = column.name
             return if field_exists?(name)
 
-            field name, Types.type_from_column(column), null: !auto_nil || column.null
+            field name, Types.type_from_column(column), null: !auto_nil || column.null,
+            authorize: -> (obj, ctx) do
+              restriction = ctx[:current_user]&.restrictions&.detect { |el|
+                (el.resource.name == name || el.resource.alias == name) &&
+                el.restriction_operation_id == "HasHelpers::RestrictionOperation::::View" &&
```

Comment:
el.restriction_operation == HasHelpers::RestrictionOperation::::View

Reply:
this will be done in another ticket, when all basic constants are added in hq-graphql through Utilities gem